server/controllers/list.js

```diff
@@ -111,18 +113,31 @@ const getArchivedListsMetaData = (req, resp) => {
   );
 };
 
+const checkIfCurrentUserVoted = (item, userId) =>
```

Comment:
I wonder if  that method should be place in here. It is not a controller in fact. It probably could be defined as instance or static method for its model. Or maybe it could be placed in utilities folder like in user's helper methods case. 
https://mongoosejs.com/docs/2.7.x/docs/methods-statics.html
https://mongoosejs.com/docs/api.html#schema_Schema-method